datapackage_pipelines_mojp/bagnowka/processors/download.py

```diff
@@ -0,0 +1,53 @@
+from datapackage_pipelines_mojp.common.processors.base_processors import BaseProcessor
+import json
+import logging
+import time
+
+
+class Processor(BaseProcessor):
+
+    def __init__(self, *args, **kwargs):
+        super(Processor, self).__init__(*args, **kwargs)
+
+    @classmethod
+    def _get_schema(cls):
+        # TODO change scheme
+        return {"fields": [{"name": "source", "type": "string"}, {"name": "collection", "type": "string"},
+            {"name": "name", "type": "string"},
+            {"name": "main_image_url_aws_s3", "type": "string"},
+            {"name": "description", "type": "string"},
+            {"name": "id", "type": "integer"},
+            {"name": "date_taken", "type": "string"},
+            {"name": "parsed_doc", "type": "object"}]}
```

Comment:
@OriHoch dates were too ununified, added a detailed description and left the type "string". will try to handle it in convert.py